lua/entities/gmod_wire_expression2/core/custom/constraintcore.lua

```diff
@@ -1,14 +1,97 @@
 E2Lib.RegisterExtension("constraintcore", false, "Allows the creation and manipulation of constraints between entities.")
 
+local Vector = Vector
+local IsValid = IsValid
+local math_min = math.min
+local math_max = math.max
+local table_insert = table.insert
+
+local cvFlags = {FCVAR_ARCHIVE}
+local maxWeld = CreateConVar( "wire_expression2_max_constraints_weld", "0", cvFlags, 0 )
```

Comment:
Nitpick and not really necessary to change but, these convar objects are just used once in the `countLookup` table, why not just have them inlined inside of it?

Reply:
No that's a good point - I was going to, but I worried that because of the indentation it would get _really_ long.
Also, `maxPerEntity` is only used outside of that table (though, I suppose I could change that if we put them all in the table).

What do you think? I'm fine with it either way.

Reply:
Think I'd prefer the ones that can be in the table to be in the table